From 8a405e433291cd48148275fdbe0dac423c43e983 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sat, 24 Jan 2026 11:47:46 +0100
Subject: [PATCH 103/N] configure: allow older autoconf

We require autoreconf to run for the build, and we use
Ubuntu 24.04 for llvm which only ships 2.71. It looks like there
wasno real reason for upstream to require it, so this should be fine.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8c07865..bfdb3ef 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,7 +12,7 @@ dnl
 # Set VERSION so we only need to edit in one place (i.e., here)
 m4_define([PYTHON_VERSION], [3.14])
 
-AC_PREREQ([2.72])
+AC_PREREQ([2.71])
 
 AC_INIT([python],[PYTHON_VERSION],[https://github.com/python/cpython/issues/])
 
